frontend/js/components/LocationField.vue

```diff
@@ -172,6 +172,11 @@
 
         this.address = newValue
         this.$emit('change', newValue)
+
+        const latlng = newValue.match(/^(-?\d+(?:\.\d+)?),+ *(-?\d+(?:\.\d+)?)$/)
+        if (latlng) {
+          this.onLatLngEntered(latlng[1], latlng[2]);
+        }
```

Comment:
Nice regexp, it seems pretty robust.

What do you think of doing this check in `onBlur()` instead? I don't think there's an issue in terms of performance with `onInput()`, just asking out of curiosity.

Reply:
If it's a blur event then you have to change focus before it triggers and with that method I often find that I'll finish typing and have a confused moment where nothing happens until I remember I need to tab off the input. Using an input event just means it happens immediately without me having to take any extra steps.

What I've got here shouldn't impact performance but we could just debounce it so it only happens once you stop typing, but in my use case we were just copy-pasting lat/lng's into the field so it didn't matter.

Reply:
Thanks for the feedback on this. I think adding a light debounce makes a lot of sense.